src/Components/TodoListItem.jsx

```diff
@@ -19,4 +20,10 @@ function TodoListItem({ title, onRemoveTodo, id }) {
   );
 }
 
+TodoListItem.propTypes = {
+  title: PropTypes.string.isRequired,
+  onRemoveTodo: PropTypes.func,
+  id: PropTypes.string.isRequired,
```

Comment:
isRequired is a very useful flag: you can provide it for all props that cannot be empty